File: TvShowApp/app/src/main/java/com/gm/gislain/tvshowapp/Model/Actor.java
package com.gm.gislain.tvshowapp.Model;

/**
 * Created by Gislain on 27/11/2016.
 */

public class Actor {

    private int id;
    private String name;

    public static final String KEY_ID = "id";
    public static final String KEY_NAME = "name";

    public Actor(){}

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: TvShowServer/Model/actor.py
class Actor(object):

    def __init__(self, id, name):
        self.id = id
        self.name = name

    def getName(self):
        return self.name

    def getId(self):
        return self.id

    def serialize(self):
        return {'id': self.id, 'name': self.name}
